jwst/assign_wcs/miri.py

```diff
@@ -422,8 +422,8 @@ def detector_to_abl(input_model, reference_files):
 
     with RegionsModel(reference_files['regions']) as f:
         allregions = f.regions.copy()
-        # Use the 50% throughput slice mask
-        regions=allregions[4,:,:]
+        # Use the 80% throughput slice mask
+        regions=allregions[7,:,:]
```

Comment:
The regression test that tests this will have to updated as well.

https://github.com/spacetelescope/jwst/blob/06d53043fb642142652c89931eef6237db3ba1a7/jwst/tests_nightly/general/miri/test_miri_steps_single.py#L174

Reply:
Done